torch/distributions/constraints.py

```diff
@@ -463,13 +463,11 @@ class _PositiveDefinite(Constraint):
     event_dim = 2
 
     def check(self, value):
-        matrix_shape = value.shape[-2:]
+        # Assumes that matrix or matrices in value are symmetric.
+        # There are also non-symmetric positive definite matrices.
```

Comment:
From Wikipedia:
>  a symmetric n x n real matrix M is said to be positive-definite ...

and then
> Some authors use more general definitions of definiteness, including some non-symmetric real matrices, or non-Hermitian complex ones.

I think for non-symmetric that would be a "generalized positive-definite" matrix 😀